studio/src/editors/merch-card-editor.js

```diff
@@ -189,13 +189,21 @@ class MerchCardEditor extends LitElement {
                 <sp-field-label for="prices">Prices</sp-field-label>
                 <rte-field
                     id="prices"
-                    inline
                     data-field="prices"
                     default-link-style="primary-outline"
                     @change="${this.updateFragment}"
-                    >${unsafeHTML(form.prices.values[0])}</rte-field
+                    >${unsafeHTML(form.prices?.values[0])}</rte-field
```

Comment:
Why this is needed? prices field is there. IMO we should not add uncertainty to code.